packages/help-center/src/components/help-center-notice.tsx

```diff
@@ -39,10 +39,16 @@ function getResponses( siteName?: string ) {
 					),
 					{
 						hosted_on_our_services: (
-							<ExternalLink href={ localizeUrl( 'https://wordpress.com/support/com-vs-org/' ) } />
+							<ExternalLink
+								href={ localizeUrl( 'https://wordpress.com/support/com-vs-org/' ) }
+								children={ null }
```

Comment:
Why is that necessary? `children` will be `undefined` by default.

Reply:
The problem is that some components require children, which is hard to support in some cases (like when interpolating)

Reply:
Makes sense, thanks.

Intuitively, I would suggest resolving this by altering the type to make the `children` prop optional, but a double-check at the `ExternalLink` code makes me think twice about it. Why do we have an external link without any text? Sounds like it could be an accessibility issue.

Reply:
Actually, it has as it used by `createInterpolateElement` function to replace the custom-defined tag with the specified component, and there is the text between the tag that is the text for the `ExternalLink`. For example,

```js
createInterpolateElement(
  '{{a}}Learn more{{/a}}',
  { a: <ExternalLink /> }
);
```

Reply:
Cool! Then (sorry for going in circles 😅 ), I'd ask the following question: what do you feel is the best in this case - passing `children={ null }`, ignoring the TS rule, or fixing the `ExternalLink` type to support an optional `children` prop`? If you picked one of those, why did you prefer that choice?

Reply:
I think the `ExternalLink` still needs the `children` so we should not make it optional. So, I'd prefer to pass `children={ null }` or even use `ts-expect-error` to bypass the rule and see whether we can fix this issue via `createInterpolateElement` as it should let TS know it will pass the children to that component. However, I'm not whether it's possible or not 😓

Reply:
Sounds good, thanks for confirming, let's keep the `children={ null }` then 👍 No need to overcomplicate things.

Reply:
I also asked about this in Slack a couple weeks ago (it's a broader issue uncovered in the react 18 update): p1684448743933079-slack-CK1QYS6P6